tests/cisco_ios/show_tacacs/cisco_ios_show_tacacs_status_fail_count.yml

```diff
@@ -0,0 +1,31 @@
+---
+parsed_sample:
+  - tacacs_server: "10.20.30.1"
+    tacacs_server_name: "TACACS_1"
+    server_port: "49"
+    socket_opens: "21752"
+    socket_closes: "21752"
+    socket_aborts: "0"
+    socket_errors: "0"
+    socket_timeouts: "0"
+    failed_connections: "52"
+    packets_sent: "23450"
+    packet_received: "23439"
+    server_status: "Alive"
```

Comment:
I believe this parsed output needs regenerated since server_status and the continuous_auth fails are still in the structured output.

Reply:
so the 3 lines at the bottom in the YML file should go ?

Reply:
I'd regenerate with the cli.py helper script.

Six lines total.

Since there are two T+ servers there are two groups of those three lines.

```
server_status: "Alive"
continous_authc_fail_count: "0"
continous_authz_fail_count: "0"
```